docs/pages/ready-made-features/comments/users-and-mentions.mdx

```diff
@@ -102,9 +102,65 @@ function Component() {
 }
 ```
 
-## Resolving user mentions suggestions
+## Adding group information to Comments
 
-Comments allows you to search for a user after typing the `"@"` character.
+Similar to users, group information is retrieved from a list of group IDs. You
+should return the same number of groups as the number of group IDs, in the same
+order.
+
+```shell
+# If this is `groupIds`
+["engineering", "design"]
+
+# Return `groups`
+[{ name: "Engineering", avatar: "https://example.com/engineering.png" }, { name: "Design", avatar: "https://example.com/design.png" }]
+```
+
+In your Liveblocks config file, you'll find a function named
+[`resolveGroupsInfo`](/docs/api-reference/liveblocks-client#resolveGroupsInfo)
+where you can return this information.
+
+```ts file="liveblocks.config.ts"
+// ...
+
+const client = createClient({
+  // ...
+  async resolveGroupsInfo({ groupIds }) {
+    // ["engineering", ...]
+    console.log(groupIds);
+
+    // Get groups from your back end
+    const groups = await __fetchGroups__(groupIds);
+
+    // [{ name: "Engineering", avatar: "https://example.com/engineering.png" }, ...]
+    console.log(groups);
+
+    // Return a list of groups
+    return groups;
+  },
+});
+```
+
+The `name` and `avatar` are required for the default components, but you can
```

Comment:
They're not strictly required, if you don't pass `name` the group ID will be displayed and `avatar` is optional too. They're handled by the default components but they're not forced